wdl-grammar/tests/parsing/imports/source.wdl

```diff
@@ -0,0 +1,8 @@
+# This is a test of import statements.
+
+version 1.1
+
+import "a.wdl"
+import "http://example.com/lib/stdlib.wdl" as stdlib
+import "c.wdl" alias Foo as Bar alias Baz as Qux 
+import "d.wdl" as d alias Foo as Bar alias Baz as Qux alias Quux as Corge
```

Comment:
Well, TIL that you can `alias` like this to resolve conflicts.

Reply:
For the record this was news to me too lol

Although I think `import "d.wdl" as d` is meaningless. That should maybe be a lint warning?

Reply:
Sounds like a good candidate for a "redundant aliasing" lint, either here in the `as` clause or for any `alias Foo as Foo`.